appleAppSiteAssociation.js

```diff
@@ -0,0 +1,76 @@
+import * as fs from "fs";
+import * as path from "path";
+
+let appId;
+
+try {
```

Comment:
remove try/catch so that this will fail and exit if something goes wrong. currently it logs